### ISSUE TITLE
Temps de réponse

### ISSUE THREAD

**Issue by ArianeFire** [LaraSpeed/LaraSpeed#24]
Le temps de réponse est très long.

![image001](https://cloud.githubusercontent.com/assets/12550052/23096431/38841996-f614-11e6-8403-547351e0f915.jpg)

**Comment by ArianeFire** [comment]
Bernard, 
Ceci est surement dû au nombre d'enregistrement de la table "rental", comme sur l'image il en existe 16039 enregistrements, donc il faudra une pagination qui est effectué en javascript par le theme "Porto Admin", je vais le customizer ça devrais gérer ce problème

![charge](https://cloud.githubusercontent.com/assets/12550052/23097062/76499130-f622-11e6-9a21-7c8114f7130b.png)

**Comment by ballesta** [comment]
Je suis d’accord sur ton analyse, il me semble que le thème Porto a plusieurs façons de gérer les table de grande taille

·        En n’envoyant que la page courante, sans Ajax, Laravel supporte cette méthode

·        En faisant du Ajax

 

Regarde les différentes méthodes, mais nous devrions pouvoir utiliser ce qu’offre Porto sans développer notre propre gestionnaire en Ajax.

 

 

 

De : Seydou Berthe [mailto:notifications@github.com] 
Envoyé : samedi 18 février 2017 22:39
À : LaraSpeed/LaraSpeed
Cc : Subscribed
Objet : Re: [LaraSpeed/LaraSpeed] Temps de réponse (#24)

 

Bernard,
Ceci est surement dû au nombre d'enregistrement de la table "rental", comme sur l'image il en existe 16039 enregistrements, donc il faudra une pagination qui est effectué en javascript par le theme "Porto Admin", je vais le customizer ça devrais gérer ce problème

 <https://cloud.githubusercontent.com/assets/12550052/23097062/76499130-f622-11e6-9a21-7c8114f7130b.png> charge

**Comment by ArianeFire** [comment]
Effectivement, j'ai combiné les deux méthodes et ça marche, 
en guise se teste : [http://163.172.174.219/LaraSpeedV2/public/rental](url)
